File: interactive/embed.py
#!/usr/bin/env python

import json
from string import Template

cssTemplate = '@import url("$base$css")'
jsTemplate = '$$.getScript("$base$js");'


base = json.load(open('package.json'))["homepage"].replace('http:', '')
manifest = json.load(open('build/asset-manifest.json'))
js = Template(jsTemplate)
open('build/embed.js', 'w').write(js.substitute(base=base, js=manifest['main.js']))

css = Template(cssTemplate)
open('build/embed.css', 'w').write(css.substitute(base=base, css=manifest['main.css']))

File: interactive/embed.js
var env = require('dotenv').config().parsed;
var manifest = require('./build/asset-manifest.json');
var fs = require('fs');

jsTemplate = `$$.getScript("$base$js");`;
const base = env["PUBLIC_URL"];
const css = manifest['main.css'];
const js = manifest['main.js'];

fs.writeFileSync('build/embed.js', `$.getScript("${base}${js}")`);
fs.writeFileSync('build/embed.css', `@import url("${base}${css}")`);

